Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -1508,19 +1515,17 @@ - (void)testAsyncOpenProgressNotifications {
 - (void)testAsyncOpenConnectionTimeout {
     [self resetSyncManager];
 
-    NSTask *task = [[NSTask alloc] init];
-    NSString *directory = [@(__FILE__) stringByDeletingLastPathComponent];
-    task.currentDirectoryPath = directory;
-    task.launchPath = @"/usr/bin/ruby";
-    task.arguments = @[[directory stringByAppendingPathComponent:@"run_baas.rb"], @"start_proxy", @"5678", @"2"];
-    [task launch];
+    TimeoutProxyServer *proxy = [[TimeoutProxyServer alloc] initWithPort:5678];
```

Comment:
I think this type needs a `stop` method. Currently this is relying on the whims of ARC to ensure that it's deallocated at the right time, but it could legally be deallocated on the line after the call to startAndReturnError.